server/aws-lsp-codewhisperer/src/language-server/chat/telemetry/chatTelemetryController.ts

```diff
@@ -35,14 +35,9 @@ interface MessageTrigger extends TriggerContext {
     followUpActions?: Set<string>
 }
 
-interface StartTrigger {
-    triggerType?: TriggerType
-    hasUserSnippet?: boolean
```

Comment:
does this mean we never logged `hasUserSnippet` field anywhere so we won't be logging it going forward?

Reply:
IIRC currently we are not using `hasUserSnippet` from the start trigger but from message trigger. (But it could be a mistake since Q would have the snippet if the first request has it.)